src/silicone/utils.py

```diff
@@ -526,3 +526,38 @@ def _construct_consistent_values(aggregate_name, components, db_to_generate):
     use["unit"] = units.iloc[0]
     use["variable"] = aggregate_name
     return pyam.IamDataFrame(use)
+
+
+def _make_weighting_series(df, weights):
+    """
+    Makes a complete list of weights for all scenarios from a dictionary of only
```

Comment:
```suggestion
    Makes a complete list of weights for all timeseries from a dictionary of only
```

Given that scenario may not be unique (because only (model, scenario) tuples define unique timeseries)?